### ISSUE TITLE
high contrast mode issues 

### ISSUE THREAD

**Issue by danielweck** [readium/readium-js-viewer#73]
"About Readium" button not visible in Windows 

Using Firefox version 30 running in Windows 7 the Readium About button is not visible in the nav bar when Windows high contrast mode is enabled. This is because a background image is used for the button face.  

Since Readium doesn't load many images it may be easier just to use a real <img> element rather that detecting high contrast mode and showing a hidden text span.  We could also add the image as :before or :after content - these work with high contrast on but do not work when images are turned off.

Bugzilla:
https://app.devzing.com/Readium/bugzilla/show_bug.cgi?id=84
